src/firmware/main/primitives/accurate_shoot.c

```diff
@@ -23,7 +23,7 @@
 static float destination[3], major_vec[2], minor_vec[2];
 // Only need two data points to form major axis vector.
 
-primitive_params_t *global_params;
+primitive_params_t* global_params;
```

Comment:
Hm... I like it the other way around `type *var` @MathewMacDougall @akhilveeraghanta what do you think?

Reply:
So this is a formatting question, so somewhat out of scope here, but I strongly prefer this. If you were to ask what the type of `global_params` is, it's a "primitive_params_t _pointer_", so I think this formatting helps convey the point that the pointer is part of the type, not the name.

Reply:
Yeah I see